get_session_request.php

```diff
@@ -0,0 +1,64 @@
+<?php
+require_once 'vendor/autoload.php';
+require_once 'config.php';
+
+use \Firebase\JWT\JWT;
+
+function get_session_request($contents)
+{
+    $parsedjson = json_decode($contents, true);
+    error_log($contents);
+
+    if (JWT_ENABLED) {
+        $jwt_pk = file_get_contents(ROOT_DIR . IRMA_SERVER_PUBLICKEY);
+        try {
+            $decoded = JWT::decode($contents, $jwt_pk, array('RS256'));
+
+            $disclosed = (array) $decoded->disclosed;
+            foreach ($disclosed as $con) {
+                foreach ($con as $attr) {
+                    if ($attr->id == ISSUER . ".gemeente.personalData.fullname" || $attr->id == ISSUER . '.pbdf.linkedin.familyname') {
+                        $fullname = $attr->rawvalue;
+                    }
+                }
+            }
+
+            if (!$fullname) {
+                $fullname = "John Doe";
+            }
```

Comment:
This handles this case only when `JWT_ENABLED` is true, but it has nothing to do with that flag; it can also occur when `JWT_ENABLED` is false.